dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -454,68 +454,136 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
      */
 
     @Test
-    public void test_MultilangContent_IndexingAllLanguages()
+    public void test_ContentInDefaultAndSecondLang_PageInDefaultLang_SiteSearchBothLangs_ShouldGetResultsForBothVersions()
             throws DotPublishingException, JobExecutionException, DotDataException, IOException, DotSecurityException, WebAssetException {
 
         boolean defaultContentToDefaultLangOriginalValue =
                 Config.getBooleanProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", false);
 
         try {
-
             Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", true);
 
             final Host site = new SiteDataGen().nextPersisted();
-            Language lang1 = APILocator.getLanguageAPI().getDefaultLanguage();
-            Language lang2 = new LanguageDataGen().nextPersisted();
+            Language defaultLang = APILocator.getLanguageAPI().getDefaultLanguage();
+            Language secondLang = new LanguageDataGen().nextPersisted();
             folder = new FolderDataGen().site(site).nextPersisted();
 
-            Contentlet contentletLang1 = TestDataUtils
-                    .getEmployeeContent(true, lang1.getId(), null, site);
+            Contentlet contentletDefaultLang = createAndPublishEmployeeContent(site, defaultLang,
+                    "catherine");
 
-            contentletLang1 = contentletAPI.find(contentletLang1.getInode(), systemUser, false);
-            contentletLang1.setStringProperty("firstName", "catherine");
-            contentletLang1 = contentletAPI.checkin(contentletLang1, systemUser, false);
+            createNewVersionAndPublishExistingEmployeeContent(secondLang, contentletDefaultLang, "catalina");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "catalina" appears 5 times in this file; the first occurrence is on line 474](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)